Formula/tfel.rb

```diff
@@ -37,6 +45,8 @@ def install
       "-Denable-diana-fea=ON",
       "-Denable-ansys=ON",
       "-Denable-europlexus=ON",
+      "-Dpython-static-interpreter-workaround=ON",
```

Comment:
What does this do?

Reply:
Why is it actually linking against python?

Reply:
Are you linking against it because you want to embed the interpreter in the library?

Reply:
@SMillerDev, no we don't want to do that, but maybe I don't fully understand the purpose of your question. Could you be more specific?

Reply:
Okay, found a better explanation than I can give. Just want to make sure that it's the correct fix for the problem. https://blog.tim-smith.us/2015/09/python-extension-modules-os-x/ has more info on the problem.

Reply:
That exactly the point of the fix.

BTW, thanks for pointing this blog post which highlights and explains the trouble that we had when porting TFEL/MFront to Mac Os. We probably will change our default policy in the future (regarding linking with -lpython) based on those guidelines, thus removing the need for this option. For now on, this is required for the current stable version.